src/components/HeaderSearch.vue

```diff
@@ -34,7 +35,11 @@ export default defineComponent({
             });
 
             isLoading.value = false;
-        });
+        };
+
+        const onInput = debounce(fetchData, 400);
```

Comment:
```suggestion
        const onInput = debounce(fetchData, 100);
```
We can shorten this time, I tested it w/100 and wasn't able to reproduce the bug but test it yourself and make sure.